src/properties/module/index.test.js

```diff
@@ -152,6 +166,37 @@ const invalidModuleConfigs = [
     },
     schema: schemaFn({ rules: { 'loader-enforce-include-or-exclude': true } }),
   },
+  {
+    // should enforce `loader` property, if object is found in `loaders` array
+    input: {
+      loaders: [{
+        test: /foo/,
+        loaders: [
+          { query: { foo: 'bar' } },
+        ],
+      }],
+    },
+    error: {
+      message: '"loaders" at position 0 does not match any of the allowed types',
```

Comment:
Unfortunately, the ability to set custom error was added in `joi@9.0.0-1` (https://github.com/hapijs/joi/blob/v9.0.0-1/API.md#anyerrorerr) and `webpack-validator` is currently using `joi@9.0.0-0`.

But even if we update it the error we can set is in the form of raw text and we have no access to additional info, like loader's index. The error would look like:

"loader should be of type "Object" or "String",

but the user wouldn't know where exactly the error occurred. 

I haven't dealt with `Joi` before, so maybe someone else knows another way to change the error message.